### PR TITLE
Adding links and doc updates to release notes

### PR DESCRIPTION
**Pull request by jseldess**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/323)

### CODE REVIEW

**Comment by jseldess** [comment]
@bdarnell, could you please regenerate the sql diagrams and functions doc?

File: timestamp.md [hunk 1]
```diff
@@ -9,7 +9,10 @@ The `TIMESTAMP` [data type](data-types.html) stores a date and time pair in UTC.
 
 ## Aliases
 
-In CockroachDB, `TIMESTAMPZ` is an alias for `TIMESTAMP`.
+In CockroachDB, the following are aliases for `TIMESTAMP`:
+
+- `TIMESTAMPZ`
```
Comment (bdarnell) [inline timestamp.md]: I don't think that's true. We have two timestamp types:
* `TIMESTAMP WITHOUT TIME ZONE`, which may be (and usually is) abbreviated `TIMESTAMP`
* `TIMESTAMP WITH TIME ZONE`, which may be (less commonly) abbreviated `TIMESTAMP TZ`

**Comment by bdarnell** [comment]
LGTM except for the timestamp part; it might be better to leave `TIMESTAMP WITHOUT TIME ZONE` out of `timestamp.md` until we can rework that page to better explain the distinction.

**Comment by jseldess** [comment]
@bdarnell, just took at quick swing at correcting `timestamp.md`. Better? If not, I can leave it out for now.

**Comment by bdarnell** [comment]
It's `TIMESTAMPTZ`, not `TIMESTAMPZ`. LGTM

**Comment by jseldess** [comment]
Whoops. Thanks for catching that, Ben.

**Pull request merged**